project/Project.scala

```diff
@@ -85,6 +87,15 @@ object Zipkin extends Build {
       "org.apache.zookeeper" % "zookeeper" % "3.4.6", // internal twitter + kafka + curator
       "org.slf4j" % "slf4j-api" % "1.6.4" // libthrift 0.5 otherwise pins 1.5.x
     )
+    // Stop publishing to local repo
```

Comment:
yeah, now that you mention that, this seems totally unnecessary and seems to override expected behavior, removing altogether.